### PR TITLE
Properly convert ScrollContainer width and height to integers

### PR DESCRIPTION
**Pull request by benedikt**
This fixes the broken scrolling for elements inside overflow: auto/scroll containers.

### CODE REVIEW

**Comment by ygongdev** [comment]
@benedikt Hi I would love to merge this, but I want to get some context on the issue.
Do you have a demo of the bug available by any chance?
I want to see if we're ok with converting it to `int` because I think the box model actually preserves the `decimals` values and use it to calculate layout

**Comment by benedikt** [comment]
@ygongdev I don't have a demo up, but the issue boils down to this: `.width` and `.height` on the results of `getComputedStyle` return a string including a `px` suffix. As a result, any calculations afterwards return `NaN`.

**Comment by ygongdev** [comment]
@benedikt got it got it. That  makes a lot of sense. What do you think about of using `parseFloat` instead to preserve the true value as much as possible?

**Comment by benedikt** [comment]
@ygongdev Yes, that should work as well. I'll update the pull request accordingly.

**Comment by benedikt** [comment]
@ygongdev I just pushed the `parseFloat` variant. 

FYI: There are a couple more `parseInt` calls in other places as well. It probably makes sense to update those eventually.

File: addon/system/scroll-container.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ export default class ScrollContainer {
       let { top, left } = this.element.getBoundingClientRect();
       this.top = top;
       this.left = left;
-      this.width = getComputedStyle(this.element).width;
-      this.height = getComputedStyle(this.element).height;
+      this.width = parseFloat(getComputedStyle(this.element).width, 10);
+      this.height = parseFloat(getComputedStyle(this.element).height, 10);
```
Comment (ygongdev) [inline addon/system/scroll-container.js]: `parseFloat` doesn't have a 2nd param. It doesn't need a radix unlike `parseInt`.

**Comment by ygongdev** [comment]
@benedikt if you want feel free to update those as well! I left a comment.

File: addon/system/scroll-container.js (hunk 1 above)
Comment (benedikt) [reply addon/system/scroll-container.js]: Done ✅

**Pull request merged**